exe/mauth-client

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
 
 $LOAD_PATH.unshift File.expand_path('../lib', File.dirname(__FILE__))
 
-require 'rubygems'
 require 'faraday'
-require 'faraday_middleware'
```

Comment:
curious to see what are the actual consequences of this, but I need to scan through all the robocop changes.
would have been easier to review in separate PRs...

Reply:
The reasoning for the the removal is explained in the PR description, do you have other concerns?

Reply:
No, I was just wanted to see the relevant changes throughout the whole PR. I'm always 👍🏼 with removing something unnecessary